### PR TITLE
Updates to LTS branch

### PR DESCRIPTION
**Pull request by ciju-nvidia**
This PR contains the changes for:
 -  Upgrade of LTS kernel to 5.10.179 
 - Correct sensor names (Bugs: #3584693)
 - Add helper function to detect ASIC hwmon existence (Bugs: #3608425)

### CODE REVIEW

File: usr/usr/bin/hw-management.sh [hunk 1]
```diff
@@ -2325,7 +2334,8 @@ do_chip_up_down()
 			exit 0
 		fi
 		chipup_delay=$(< $config_path/chipup_delay)
-		if [ ! -d /sys/bus/i2c/devices/"$bus"-"$i2c_asic_addr_name" ]; then
+		retry_helper find_asic_hwmon_path 0.2 3 "chip hwmon object" /sys/bus/i2c/devices/"$bus"-"$i2c_asic_addr_name"
```
Comment (vadimp-nvidia) [inline usr/usr/bin/hw-management.sh:2337]: I tried to compare against r-build-bsp-02:/hdd/build/crajank/hw_mgmt/20_11_2023/hw-mgmt/usr/usr/bin/hw-management.sh.
There you have line:
retry_helper find_asic_hwmon_path 0.2 3 "chip hwmon object" /sys/bus/i2c/devices/"$bus"-"$i2c_asic_addr_name"/hwmon
Here same line with no 'hwmon' at the end.

Comment (ciju-nvidia) [reply usr/usr/bin/hw-management.sh]: As per our discussion, updated the changes in the next version.

**Review by sholeksandr — APPROVED** [review]

**Comment by ciju-nvidia** [comment]
RERUN

**Comment by ciju-nvidia** [comment]
RERUN

**Comment by ciju-nvidia** [comment]
RERUN

**Pull request merged**